frontend/script.js

```diff
@@ -0,0 +1,30 @@
+let type = "WebGL"
+if(!PIXI.utils.isWebGLSupported()){
+    type = "canvas"
+}
+PIXI.utils.sayHello(type);
+
+window.onload = start;
+
+function start() {
+    //Create a Pixi Application
```

Comment:
I think it's obvious, we can remove this comment

Reply:
ok